File: sdk/javascript-closure-angular/API/Client/TimetableList.js
goog.provide('API.Client.TimetableList');

/**
 * @record
 */
API.Client.TimetableList = function() {}

/**
 * @type {!string}
 * @export
 */
API.Client.TimetableList.prototype.errorcode;

/**
 * @type {!string}
 * @export
 */
API.Client.TimetableList.prototype.errormessage;

/**
 * @type {!number}
 * @export
 */
API.Client.TimetableList.prototype.numberofresults;

/**
 * @type {!string}
 * @export
 */
API.Client.TimetableList.prototype.route;

/**
 * @type {!string}
 * @export
 */
API.Client.TimetableList.prototype.stopid;

/**
 * @type {!string}
 * @export
 */
API.Client.TimetableList.prototype.timestamp;

/**
 * @type {!Array<!API.Client.Timetable>}
 * @export
 */
API.Client.TimetableList.prototype.results;

File: sdk/async-scala/src/main/scala/io/swagger/client/model/TimetableList.scala
package io.swagger.client.model

import org.joda.time.DateTime
import java.util.UUID


case class TimetableList (
  errorcode: Option[String],
errormessage: Option[String],
numberofresults: Option[Integer],
route: Option[String],
stopid: Option[String],
timestamp: Option[String],
results: Option[List[Timetable]]
)
